pkg/minikube/cluster/cluster.go

```diff
@@ -192,6 +192,13 @@ func preCreateHost(config *cfg.MachineConfig) error {
 				See https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm2-driver for more information.
 				To disable this message, run [minikube config set WantShowDriverDeprecationNotification false]`)
 		}
+	case "parallels":
+		if viper.GetBool(cfg.ShowDriverDeprecationNotification) {
+			fmt.Fprintln(os.Stderr, `WARNING: The parallels driver is now deprecated and support for it will be removed in a future release.
```

Comment:
Adding a deprecated driver isn't a good user experience. Can this PR add the non-deprecated one?

Reply:
The code section is cleared of change.